include/CppUTest/CppUTestConfig.h

```diff
@@ -216,8 +218,6 @@ typedef struct { char dummy[16]; } cpputest_ulonglong;
 typedef struct { char dummy[8]; } cpputest_longlong;
 typedef struct { char dummy[8]; } cpputest_ulonglong;
 #endif
-#define CPPUTEST_LONGLONG_DEFAULT cpputest_longlong()
-#define CPPUTEST_ULONGLONG_DEFAULT cpputest_ulonglong()
```

Comment:
These are essential if you want coverage in the case where `CPPUTEST_USE_LONG_LONG` is not defined (it will not compile without them).  If you don't care about coverage in that case then many tests could be removed to eliminate use of this value, but as it stands it's not safe to remove these.